apps/examples/wc-vanilla/index.html

```diff
@@ -34,7 +34,7 @@
         <script type="module" src="/main.js"></script>
         <script type="module" src="./node_modules/@justeattakeaway/pie-button"></script>
         <script type="module" src="./node_modules/@justeattakeaway/pie-icon-button"></script>
-        <script type="module" src="./node_modules/@justeattakeaway/pie-icons-webc/icons/IconClose"></script>
-        <script type="module" src="./node_modules/@justeattakeaway/pie-icons-webc/icons/IconSearch"></script>
+        <script type="module" src="./node_modules/@justeattakeaway/pie-icons-webc/dist/icons/IconSearch.js"></script>
```

Comment:
I dont think there is a icons folder inside the dist directory, i had a similar issue. Are you sure this works?

Reply:
tagged in slack for more details
but the latest updates of the icons-webc package requires this change otherwise, the app won't build